data/fields/seamark/name.json

```diff
@@ -0,0 +1,5 @@
+{
+    "key": "seamark:name",
```

Comment:
there are few problems here: using weird duplicate of `name` should  not be promoted, in my opinion

if current ship berth tagging schema requires it then for me it should block its support in editors

also, https://wiki.openstreetmap.org/wiki/Tag:seamark:type=berth claims that `seamark:berth:name` should be used (which manages to be an even weirder idea)

Reply:
If it is name then `name` would work - is it supported there?

If it is in fact reference then `seamark:berth:name` is even sillier.

> If you want iD to start encouraging a different tagging practice, those apps will need to be updated too

Maybe. In my opinion it should be prerequisite to supporting this tagging scheme anywhere (and while at it, maybe getting rid of `seamark:` prefix from key would be doable?)